ci/qa_models/vllm_opt/model.json

```diff
@@ -0,0 +1,5 @@
+{
+    "model":"facebook/opt-125m",
+    "disable_log_requests": "true",
+    "gpu_memory_utilization": 0.5
```

Comment:
You can drop "gpu_memory_utilization" from our test model. I saw some failures when using this limit on my machine.

```
100%|################################################################################################################################| 441/441 [00:00<00:00, 1.10MB/s]
Downloading pytorch_model.bin: 100%|#####################################################################################################################################| 251M/251M [00:02<00:00, 114MB/s]
INFO 10-08 07:11:33 llm_engine.py:205] # GPU blocks: 19794, # CPU blocks: 7281
E1008 07:11:39.335147 94 backend_model.cc:634] ERROR: Failed to create instance: Stub process 'vllm_opt_0_0' is not healthy.
E1008 07:11:39.345423 94 model_lifecycle.cc:621] failed to load 'vllm_opt' version 1: Internal: Stub process 'vllm_opt_0_0' is not healthy.
```

Reply:
Updated 👍